### PR TITLE
fix: 行列容器组件纵向分割大于2的时候报错

### LINKED ISSUE #469: 🐛 [Bug]: 行列容器组件纵向分割大于2的时候报错
### Environment

dev

### Version

20

### Version

latest

### Link to minimal reproduction

行列容器组件纵向分割大于2的时候报错

### Step to reproduce

行列容器组件纵向分割大于2的时候报错

### What is expected

_No response_

### What is actually happening

_No response_

### What is your project name

最新项目

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by gene9831**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

fix #469

### What is the new behavior?
col-splitter

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/canvas/src/components/container/CanvasDivider.vue [hunk 1]
```diff
@@ -129,7 +129,10 @@ export default {
           ]
         } else {
           // 已经切割过了，直接加一行
-          schema.children.push(...extend(true, {}, ROW_SNIPPET))
+          schema.children.push({
+            ...extend(true, {}, ROW_SNIPPET),
```
Comment (chilingling) [inline packages/canvas/src/components/container/CanvasDivider.vue]: 直接  `schema.children.push(extend(true, {}, ROW_SNIPPET))` 是不是就可以了？

Comment (gene9831) [reply packages/canvas/src/components/container/CanvasDivider.vue]: 思路是对的，还需要加上`children`属性
```js
schema.children.push({
  ...extend(true, {}, ROW_SNIPPET),
  children: [extend(true, {}, COL_SNIPPET)],
})
```

Comment (chilingling) [reply packages/canvas/src/components/container/CanvasDivider.vue]: `ROW_SNIPPET` 的 children 属性默认自带  `COL_SNIPPET`

Comment (gene9831) [reply packages/canvas/src/components/container/CanvasDivider.vue]: 已修改

**Review by chilingling — DISMISSED** [review]

**Review by chilingling — APPROVED** [review]

**Pull request merged**